commands/mongo/mongo-dump-database.md

```diff
@@ -0,0 +1,10 @@
+### Dump database from MongoDB
+
+`mongodump --db={{DB_NAME}} --out={{DESTINATION_DIRECTORY}}`
+
+- <b>DB_NAME: </b> Database's name.
+- <b>DESTINATION_DIRECTORY: Directory to save the output files. However, it is an optional parameter. By default, the command saves the registers in the current path.
```

Comment:
`<b>DESTINATION_DIRECTORY: ` => `<b>DESTINATION_DIRECTORY: </b>`

close the `<b>` tag here